packages/core/http/core-http-router-server-internal/src/request.ts

```diff
@@ -206,13 +217,17 @@ export class CoreKibanaRequest<
     } as unknown as KibanaRequestRouteOptions<Method>; // TS does not understand this is OK so I'm enforced to do this enforced casting
 
     return {
-      path: request.path,
+      path: request.path ?? '/',
       method,
       options,
     };
   }
 
-  private getAuthRequired(request: Request): boolean | 'optional' {
+  private getAuthRequired(request: RawRequest): boolean | 'optional' {
+    if (!isRealRawRequest(request)) {
```

Comment:
nit: Should we use the fake validation instead?
```suggestion
    if (isFakeRawRequest(request)) {
```